packages/playground/src/components/Editor/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
+import MonacoEditor from 'react-monaco-editor';
+import exampleCode from './example-code';
+
+export const Editor = () => {
+  const [code, setCode] = useState(exampleCode);
+  const [isDark, setIsDark] = useState(true);
+  const [showMinimap, setShowMinimap] = useState(true);
+  const [showNumbering, setShowNumbering] = useState(true);
+
+  const handleChange = (newValue: string) => {
```

Comment:
I'd suggest to always explicitly define the return type of a function. We could even enforce this with an ESlint rule in the future (https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md)